CRM/Utils/Check/Component/Env.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+/** @noinspection PhpUnused */
```

Comment:
Curious what this is needed for

Reply:
it tells phpstorm not to see the functions in the file as unused - but I can't get it past the style police so removing